### DISCUSSION TITLE
Dynamic (persistent) filter with presets?

### DISCUSSION THREAD

**Discussion by gerhardmpl** [Budibase/budibase#5354] [Ideas]
Is there a way to define presets for dynamic filters that apply to all user? I have a list of work items that are either active or not (boolean table field). Would be helpful to have the dynamic filter generaly show the active once for daily work but allowing the user to show all tasks by deselecting the filter. Persistent filter would be a plus, because one has to redefine the filter after editing a row and going back to the table screen.

**Comment by melohagan** [comment]
This is a good idea, but you could also use filter bindings in combination with App State (which can be persisted) to get the desired result.

Docs:
 - https://docs.budibase.com/docs/searchfilter-data#filtering-data-from-the-front-end
 - https://docs.budibase.com/docs/app-state
